tests/__init__.py

```
@@ -261,7 +278,7 @@ def loadDocstringsFromModule(module: types.ModuleType) -> TestSuite:
 		>>>
 
 	"""
-	if not module:
+	if not module:  # pragma: no branch
```

Flake8 (7.2.0)

[warning] indentation contains tabs

(W191)